packages/discord.js/src/structures/Guild.js

```diff
@@ -1197,6 +1197,17 @@ class Guild extends AnonymousGuild {
     return this;
   }
 
+  /**
+   * Sets whether this guild's invites are disabled.
+   * @param {boolean} [disabled=true] Whether the invites are disabled
+   * @returns {Promise<Guild>}
+   */
+  async disableInvites(disabled = true) {
+    const features = this.features.filter(feature => feature !== GuildFeature.InvitesDisabled);
+    if (disabled) features.push(GuildFeature.InvitesDisabled);
```

Comment:
Wouldn't this be more readable by making the features variable a set and then adding/removing the feature based on the param, and then converting it to an array?

Reply:
That would probably not be as efficient, and this code isn't particularly very complicated to understand either.